src/pages/komodo-defi-framework/api/common_structures/index.mdx

```diff
@@ -120,6 +120,40 @@ Where the value indicates:
   ```
 </CollapsibleSection>
 
+### EventStreamConfig
+
+The `EventStreamConfig` object defines which events will be streamed to the client:
+
+| Parameter                      | Type   | Description                                                                            |
+| ------------------------------ | ------ | -------------------------------------------------------------------------------------- |
+| access\_control\_allow\_origin | string | Defines CORS whitelist. Use "\*" to allow fromi any origin.                            |
+| active\_events                 | object | Events to be streamed to the client, along with configuration defineing frequency etc. |
+| worker\_path                   | string | WASM only. Path to a custom `worker.js` file.                                          |
+
+Active events are defined as follows:
+
+*   NETWORK: Network status changes. Requires `stream_interval_seconds` configuration.
+*   COIN\_BALANCE: Balance changes. Does not require `stream_interval_seconds` configuration, as the event is triggered by the balance change.
```

Comment:
There is another event type called `HEARTBEAT` (see https://github.com/KomodoPlatform/komodo-defi-framework/issues/2042).

Reply:
cc @smk762  (in case if you missed this note).